src/Android/Xamarin.Android/Samples/Data/EditFeatureLinkedAnnotation/readme.metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "category": "Data",
+    "description": "Edit feature attributes which are linked to annotation through an expression.",
+    "formal_name": "EditFeatureLinkedAnnotation",
+    "ignore": false,
+    "images": [
+        "EditFeatureLinkedAnnotation.jpg"
+    ],
+    "keywords": [
+        "annotation",
+        "attributes",
+        "feature-linked annotation",
+        "features",
+        "fields"
+    ],
+    "nuget_packages": {
+        "Esri.ArcGISRuntime": "100.9.0",
```

Comment:
Script may need to be updated for 100.10

Reply:
I will do this in a separate PR. It will change a lot of files.